AY2021S2-CS2103-T14-1/tp
# Enforce strict datetime parsing

## What this does
Closes #213 by changing from default `ResolverStyle.Smart` to `Strict`

## How to test
1. run `iadd r/01-234 d/Something exploded t/2020/2/30 12:00am`, it should fail and show the constraints message
2. run `iadd r/01-234 d/Something exploded t/2020/2/29 12:00am`, it should work
